### PR TITLE
need to clear input fields to avoid over writing when there is a text…

### PR DESCRIPTION
**Pull request by yohannesalazar**
… already in the field from previous test

### CODE REVIEW

**Comment by yohannesalazar** [comment]
touchui image search input field need to CLEAR before setting the field with value, (helps not to overwrite if there is an existing text from previous test ).

File: bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/DialogConfigurer.java [hunk 1]
```diff
@@ -80,6 +80,7 @@ public DialogField getDialogField(WebElement parentElement, String label, String
    */
   public DialogField getDialogField(WebElement parentElement, String type) {
     WebElement scope = parentElement.findElement(By.tagName("input"));
+    scope.clear();
```
Comment (kkarolk) [inline bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/DialogConfigurer.java]: The getDialogFIeld method should't clear any field. I would suggest to clear a field in setValue method in field related class, e.g.  Image from com.cognifide.qa.bb.aem.touch.pageobjects.touchui.dialogfields

**Comment by yohannesalazar** [comment]
Hi Karol .. thanks done so .. sorry for that

**Comment by yohannesalazar** [comment]
pushed again with no change for merging

**Pull request merged**